assets/app/scripts/services/applicationGenerator.js

```diff
@@ -56,7 +56,7 @@ angular.module("openshiftConsole")
             {
               containerPort: parseInt(parts[0]), 
               name: input.name + "-" + parts[1] + "-" + parts[0],
-              protocol: parts[1]
```

Comment:
should we be uppercasing here, or just pushing an uppercase TCP into `parts` on line 53?